src/modules/returns/void-returns.js

```diff
@@ -0,0 +1,31 @@
+const { logger } = require('../../logger');
+const { repo } = require('../../modules/returns/returns');
+
+/**
+ * For a given licence number with a list of valid return ids, makes any other
+ * returns ids for the licence id have a status of void.
+ * @param {Object} request The HAPI request containng a valdated payload
+ */
+const postVoidReturns = async request => {
+  const { regime, licenceType, licenceId, validReturnIds } = request.payload;
+  const filter = {
+    regime,
+    licence_type: licenceType,
+    licence_ref: licenceId,
+    source: 'NALD',
```

Comment:
Not sure `source` needs to be in the filter - we may need to void WRLS originated returns in future once we are in control of the returns leg